### DISCUSSION TITLE
Work with Local Repository, dev files in WSL2

### DISCUSSION THREAD

**Discussion by Because789** [sveltia/sveltia-cms#101] [Show and tell]
Ok, I feel very, very stupid now, but after like 5h of trying I figured out my issue, so I'm gonna leave the solution here, if anyone else runs into the same thing.

## The problem 

I'm a hobbyist developer and I do my stuff in this environment:

- Windows 11 Pro 23H2
- WSL 2.0.14.0 with Ubuntu 22.04
- Docker Engine 25.0.3 (no Docker Desktop) 
- VS Code on Windows with the Remote Development extensions
- Edge on Windows for testing in Browser
- Dev files are on the file system of the Ubuntu instance

I wanted to try the awesome [Astro  Starter template](https://github.com/zankhq/astro-starter) which uses Sveltia CMS. I cloned it to my dev folder on Ubuntu, set up a dev container and got it running on `localhost:4321` in no time. When I went to `localhost:4321/admin/index.html`, I got the _Work with Local Repository_ button as expected and [described here](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#work-with-a-local-git-repository), great!

But when I clicked on the button, unlike in the Windows Explorer:

![ViewInExplorer](https://github.com/sveltia/sveltia-cms/assets/867257/33dcb0a0-0cba-4a04-9fe3-0ef7f95abac6)

There was no Linux file system available:

![ViewInFolderChooser](https://github.com/sveltia/sveltia-cms/assets/867257/5afc91f3-39bf-4d2b-932e-9fdc2a5626df)

I figured, that was quite logical and it seemed to fit [the note on WSL in the README.md](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#work-with-a-local-git-repository).

## The solution

Without getting in all the details on how I wasted my time the next hours (I even set up an Ubuntu bear metal server, where it worked beautifully), it turns out I'm just stupid and the WSL integration is just great and it works out of the box without changing anything, you just have to click the right dropdown:

![DropdownOpen](https://github.com/sveltia/sveltia-cms/assets/867257/9377e04d-c183-410a-870e-813ba1d5bee4)

After choosing Linux here, I was able to choose my astro-starter folder on the Ubuntu file system and I was able to edit files in Sveltia without issues. Hope that helps others to avoid headaches...

**Comment by kyoshino** [comment]
Thanks for your insights, @Because789! Since this is a bug not in Sveltia CMS, I’ll convert the issue to a discussion, and add a link to it from README.
